envs/local_test.go

```diff
@@ -98,3 +107,261 @@ func (s *LocalSuite) TestRelationships(c *C) {
 		"PGHOST":                 "127.0.0.1",
 	})
 }
+
+func (s *LocalSuite) TestGenericContainer(c *C) {
+	ctx := context.Background()
+
+	composeProject := os.Getenv("COMPOSE_PROJECT_NAME")
```

Comment:
looks to me that it would be more realistic (and easier to debug?) to use their Docker Compose module: https://golang.testcontainers.org/features/docker_compose/